pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/entities/Badge.java

```diff
@@ -0,0 +1,79 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.entities;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * Entity class for badge object.
+ *
+ * @author Michal Mokros 456442
+ */
+@Entity(name = "Badge")
+@Table(name = "TABLE")
+public class Badge {
+
+    @Id
+    @GeneratedValue(strategy =  GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany
```

Comment:
I believe this side of the relationship gets the @ ManyToOne annotation. Make sure its lazy loaded. And knows what the join column is/will be.